src/extension/app/components/plugin/plugin-action-bar.js

```diff
@@ -352,7 +352,7 @@ export class PluginActionBar extends ConnectedElement {
 
     systemPlugins.push(html`
       <div class="logo">
-        ${ICONS.ADOBE_LOGO}
+        ${ICONS.SIDEKICK_LOGO}
```

Comment:
I decided to rename this because, technically, it has never been the Adobe logo.